### ISSUE TITLE
Error: ENOENT: no such file or directory, mkdir '/home/travis/build/ai/size-limit/temp/chrome'

### ISSUE THREAD

**Issue by ai** [mbalabash/estimo#11]
I tried new Estimo 1.1.1 in Size Limit, but got the error:

```
10.31s$ node ./node_modules/estimo/scripts/findChrome.js && npx jest test/get-running-time.test.js
Local Chrome version is not suitable
Downloading Chromium r672088...
ERROR: Failed to download Chromium r672088!
Error: ENOENT: no such file or directory, mkdir '/home/travis/build/ai/size-limit/temp/chrome'
  -- ASYNC --
    at BrowserFetcher.<anonymous> (/home/travis/build/ai/size-limit/node_modules/puppeteer-core/lib/helper.js:111:15)
    at downloadChromium (/home/travis/build/ai/size-limit/node_modules/estimo/scripts/findChrome.js:190:50)
    at findChrome (/home/travis/build/ai/size-limit/node_modules/estimo/scripts/findChrome.js:249:30)
    at processTicksAndRejections (internal/process/task_queues.js:85:5) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'mkdir',
  path: '/home/travis/build/ai/size-limit/temp/chrome'
}
(node:5778) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'executablePath' of null
    at findChrome (/home/travis/build/ai/size-limit/node_modules/estimo/scripts/findChrome.js:250:83)
```

https://travis-ci.org/ai/size-limit/jobs/563616800

/cc @mbalabash @creazero

**Comment by mbalabash** [comment]
@ai  
I think problem right here:
```js
const chromeTempPath = path.join(process.cwd(), 'temp', 'chrome')
const chromeConfigPath = path.join(process.cwd(), 'chrome.json')
```

I will fix it today.

**Comment by mbalabash** [comment]
Should be fixed in `estimo@1.1.2`

**Comment by ai** [comment]
Yeap. Thanks
